packages/sdk-components-react/src/vimeo-preview-image.ws.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import {
+  propsOverrides as imagePropsOverrides,
+  propsMeta as imagePropsMeta,
+  meta as imageMeta,
+} from "./image.ws";
+import { props } from "./__generated__/vimeo-preview-image.props";
+
+export const meta: WsComponentMeta = {
+  ...imageMeta,
+  category: "hidden",
+  label: "Preview Image",
+  requiredAncestors: ["Vimeo"],
+};
+
+export const propsMeta: WsComponentPropsMeta = {
+  props: { ...props, ...imagePropsOverrides },
```

Comment:
Props are 100% same. Just reexport image props without exposing these overrides.

Reply:
I was thinking that this will be a problem if someone changes the interface of that components, it would be very confusing to not see the changes and then find out its not using the actual props of that component

Reply:
another option to just copy/paste that bit and avoid dependency